nilearn/reporting/glm_reporter.py

```diff
@@ -712,6 +720,8 @@ def _make_stat_maps_contrast_clusters(stat_img, contrasts_plots, threshold,
         If nothing is specified, the MNI152 template will be used.
         To turn off background image, just pass "bg_img=False".
 
+    %(cut_coords)s
```

Comment:
Note that this won't render because the function isn't decorated with `fill_doc`.

Reply:
Is it a problem since the function is only used internally ?

Reply:
It is true that since the function is private, its docstring won't be rendered in the public API documentation.
But developers might still needs to take a look at its documentation somehow, so we should either copy paste the documentation of `cut_coords`, or use the `fill_doc` decorator such that the `__doc__` attribute of the function gets filled properly (this will then render properly in jupyter notebooks with the `?` command for instance instead of just printing `%(cut_coords)s`).